common/metrics/tags.go

```diff
@@ -46,6 +46,7 @@ const (
 	workflowType  = "workflowType"
 	activityType  = "activityType"
 	commandType   = "commandType"
+	serviceType   = "service_type"
```

Comment:
Rename to serviceName = "service_name" ?
We already use ServiceName in many places for this same purpose, we should try to keep consistent here as well.